src/demo-app/button/button-demo.scss

```diff
@@ -0,0 +1,15 @@
+
+// Helps fonts on OSX looks more consistent with other systems
```

Comment:
Also mention that this isn't part of the library css due to performance concerns.